modules/system/classes/extensions/PluginVersionManager.php

```diff
@@ -17,25 +21,18 @@
  * @package winter\wn-system-module
  * @author Alexey Bobkov, Samuel Georges
  */
-class VersionManager
+class PluginVersionManager
```

Comment:
Is the only reason this is a separate class to avoid having the PluginManager being too long or is it ever used by anything else?

Reply:
It is used by some other parts of the system but only via the plugin manager, i.e. `PluginManager::instance()->versionManager()`. It could probably be merged in but `PluginManager` is already chonk and confusing...